integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/MessageSender.java

```diff
@@ -110,6 +114,38 @@ public PutMessageResponseType sendMessage(EDUCore message) {
         return createOkResponse();
     }
 
+    public void sendMessage(ConversationResource conversation) throws MessageContextException, IOException {
+        MessageContext messageContext = createMessageContext(conversation);
+
+        EduDocument edu;
+        try {
+            edu = standardBusinessDocumentFactory.create(conversation, messageContext);
+            log.info("EduMessage created from ConversationResource");
+        } catch (MessageException e) {
+            log.error("Failed creating EduMessage from ConversationResource", e);
+            return;
+        }
+
+        Transport t = transportFactory.createTransport(edu);
+        t.send(context, edu);
+
+        log.info("ConversationResource sent");
+    }
+
+    private MessageContext createMessageContext(ConversationResource conversation) throws MessageContextException {
+        if (isNullOrEmpty(conversation.getReceiverId())) {
+            throw new MessageContextException(StatusMessage.MISSING_RECIEVER_ORGANIZATION_NUMBER);
+        }
+
+        MessageContext context = new MessageContext();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "context" which hides the field declared at line 43. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)